src/resource/solidDataset.ts

```diff
@@ -500,13 +500,15 @@ export async function deleteSolidDataset(
  *
  * @param url URL of the empty Container that is to be created.
  * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters).
+ * @param solidDataset Optional parameter - if provided we use this dataset as the body of the HTT request, meaning it's data is included in the Container resource.
  * @since 0.2.0
  */
 export async function createContainerAt(
   url: UrlString | Url,
   options: Partial<
     typeof internal_defaultFetchOptions
-  > = internal_defaultFetchOptions
+  > = internal_defaultFetchOptions,
+  solidDataset?: SolidDataset
```

Comment:
```suggestion
    initialContent: SolidDataset
  > = internal_defaultFetchOptions,
```
Since an `options` object is already passed, the additional optional parameter should be added to this options object, rather than added as an additional parameter. This way, the following would be used: `createContainerAt(someUrl, { initialContent: someDataset})` rather than `createContainerAt(someUrl, undefined, someDataset)`